torch/distributed/_tools/memory_tracker.py

```diff
@@ -147,26 +143,41 @@ def summary(self, top: int = 20) -> None:
         print("------------------------------------------------")
 
     @no_type_check
-    def show_traces(self) -> None:
-        """
-        Show the traces of ``memory_allocated``, ``memory_active`` and ``memory_reserved`` at
-        operator level and the marker 'fw_bw_boundary' at the boundary of forward pass
-        and backward pass.
-        """
+    def show_traces(self, path: str = "") -> None:
         import matplotlib.pyplot as plt
 
-        y_1 = [mb for (name, mb) in self.memories_allocated.values()]
-        y_2 = [mb for (name, mb) in self.memories_active.values()]
-        y_3 = [mb for (name, mb) in self.memories_reserved.values()]
-        min_val = min(y_1 + y_2 + y_3)
-        max_val = max(y_1 + y_2 + y_3)
-        x = list(i for i in range(len(y_1)))
-        fig = plt.figure(figsize=(16, 8))
-        plt.plot(x, list(y_1), label="memory_allocated")
-        plt.plot(x, list(y_2), label="memory_active")
-        plt.plot(x, list(y_3), label="memory_reserved")
+        if path != "":
+            self.load(path)
+
+        y_1 = [gb for (name, gb) in self.memories_allocated.values()]
+        y_2 = [gb for (name, gb) in self.memories_active.values()]
+        y_3 = [gb for (name, gb) in self.memories_reserved.values()]
+        min_val = min(y_1 + y_2 + y_3) * 0.9
+        max_val = max(y_1 + y_2 + y_3) * 1.1
+        x = list(range(len(y_1)))
+        #
+        plt.figure(1)
```

Comment:
Curious: What changed in terms of the plot here? It looks like you split the plot into subplots?